controllers/dockermachine_controller.go

```diff
@@ -143,8 +178,6 @@ func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		return ctrl.Result{}, err
```

Comment:
If using `defer` for the patch above, this `Update` can be removed and you can do a `return r.create(ctx, cluster, machine, dockerMachine)` above.